sklearn/ensemble/bagging.py

```diff
@@ -33,6 +33,33 @@
 MAX_INT = np.iinfo(np.int32).max
 
 
+def _draw_indices(random_state, bootstrap, n_pop, n_samp):
```

Comment:
And probably "generate" rather than "draw", unless there's a substantial difference in meaning?